oaknational/Oak-Web-Application
# All years in mobile header and unit modal

## Description
Adds all-years alternative text in mobile header and unit modal

## Issue(s)

Fixes `CUR-959`

## How to test
 - Enable cycle 2 and test for correct text in mobile header and unit modal.
 - Check for regressions in cycle 1
